.github/workflows/sycl-linux-precommit.yml

```diff
@@ -72,6 +72,56 @@ jobs:
           else
             echo 'arc_tests="Matrix/"' >> "$GITHUB_OUTPUT"
           fi
+
+  split_test_build:
+    needs: [build]
+    if: ${{ always() && !cancelled() && needs.build.outputs.build_conclusion == 'success' }}
+    uses: ./.github/workflows/sycl-linux-run-tests.yml
+    with:
+      name: Build e2e tests
+      runner: '["Linux", "build"]'
+      image: ghcr.io/intel/llvm/ubuntu2204_intel_drivers:alldeps
+      image_options: -u 1001
+      target_devices: opencl:cpu
```

Comment:
Is it possible to skip this?

Reply:
In what scenarios should we skip this? Im guessing you are suggesting adding an if statement for cases in which we dont want to run this? im not too clear on this.

Reply:
Sorry for not being clear, I meant skip/remove that line.

Reply:
yeah unclear to me why we need to set target devices as well